test/combineAllP.js

```diff
@@ -4,23 +4,34 @@ const always     = require('ramda/src/always')
 
 const { combineAllP } = require('..')
 
-const whatevs = combineAllP([
-  always(Promise.resolve({ foo: 1 })),
-  always(Promise.resolve({ bar: 2 })),
-])
-
 describe('combineAll', () => {
-  const res = property()
+  context('when partially applied', () => {
+    const res = property()
+
+    beforeEach(() =>
+      combineAllP([
+        always(Promise.resolve({ foo: 1 })),
+        always(Promise.resolve({ bar: 2 })),
+      ])({ baz: 3 }).then(res)
+    )
+
+    it('combines the results of all functions', () =>
+      expect(res()).to.eql({ foo: 1, bar: 2, baz: 3 })
+    )
+  })
```

Comment:
Without pulling in chai-as-promised or using async/await, this is the best I can think of for testing promise-returning functions in different ways

Reply:
Oh, I guess I could use `done`, or there's even another way...

Reply:
Is there a problem with using `res()` like this? This works for me & matches precedent.